airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -112,7 +112,7 @@
 - name: Chargebee
   sourceDefinitionId: 686473f1-76d9-4994-9cc7-9b13da46147c
```

Comment:
sourceDefinitionId has been removed in https://github.com/airbytehq/airbyte/pull/10302

Reply:
only the file right?